web/react/utils/client.jsx

```diff
@@ -849,3 +849,24 @@ module.exports.updateValetFeature = function(data, success, error) {
 
     module.exports.track('api', 'api_teams_update_valet_feature');
 };
+
+module.exports.isEmailEnabledSynchronous = function() {
+    var enabled = false;
+
+    $.ajax({
+        async: false,
```

Comment:
The async:false method for ajax is deprecated by JQuery. It still works but might at any time not and leads to errors popping up on our screen. Is there any way you could refactor this to be asynchronous?